extensions/amp-brightcove/0.1/amp-brightcove.js

```diff
@@ -189,6 +198,16 @@ class AmpBrightcove extends AMP.BaseElement {
       this.playing_ = false;
     }
 
+    if (data.ct) {
```

Comment:
It seems like closure isn't too happy with your dot access on `data`. That's why it is failing [Travis](https://travis-ci.org/ampproject/amphtml/jobs/471296017)
```
...Compiler issues for check-types.js:
extensions/amp-brightcove/0.1/amp-brightcove.js:201: ERROR - Cannot do '.' access on a dict
    if (data.ct) {
             ^^
extensions/amp-brightcove/0.1/amp-brightcove.js:202: ERROR - Cannot do '.' access on a dict
      this.currentTime_ = data.ct;
                               ^^
extensions/amp-brightcove/0.1/amp-brightcove.js:204: ERROR - Cannot do '.' access on a dict
    if (data.pr) {
             ^^
extensions/amp-brightcove/0.1/amp-brightcove.js:205: ERROR - Cannot do '.' access on a dict
      this.playedRanges_ = data.pr;
                                ^^
extensions/amp-brightcove/0.1/amp-brightcove.js:207: ERROR - Cannot do '.' access on a dict
    if (data.dur) {
             ^^^
extensions/amp-brightcove/0.1/amp-brightcove.js:208: ERROR - Cannot do '.' access on a dict
      this.duration_ = data.dur;
                            ^^^
```

You can run type check locally via `gulp check-types` to verify fix.

Reply:
Got it. Changed to bracket notation.